lib/ruby-debug-ide/xml_printer.rb

```diff
@@ -380,9 +384,37 @@ def max_compact_name_size
       50
     end
 
+    def inspect_with_allocation_control(slice, memory_limit)
+      x = Thread.current
```

Comment:
Can we conceive a better name for this variable? Like `current_thread`, `start_thread` or alike